src/graphql-server/resolvers.js

```diff
@@ -19,13 +20,21 @@ export const resolvers = {
 
     happs: () => promiseMap(HhaDnaInterface.happs.all(), getHappDetails),
 
+    hostPricing: () => HhaDnaInterface.hostPricing.get(),
+
+    holofuelWaitingTransactions: () => HoloFuelDnaInterface.transactions.allWaiting(),
+
+    holofuelActionableTransactions: () => HoloFuelDnaInterface.transactions.allActionable(),
+
+    holofuelCompleteTransactions: () => HoloFuelDnaInterface.transactions.allComplete(),
+
+    holofuelLedger: () => HoloFuelDnaInterface.ledger.all(),
```

Comment:
this endpoint should probably be called `get` rather than `all`. The pattern is `all` endpoints return an array of objects of the same type, while `get` endpoints return a single object.

Reply:
Good point.  I was thinking of it as 'all details' and was therefore going back and forth between the two, but the data struct is still an obj, so I like having that be the distinction. :+1: